wandb/sdk/wandb_artifacts.py

```diff
@@ -935,6 +938,58 @@ def _file_url(
         else:
             raise Exception(f"unrecognized storage layout: {storage_layout}")
 
+    def s3_multipart_file_upload(
+        self,
+        file_bytes: List[bytes],
```

Comment:
This might be a big issue. I don't want to rely on a client being able to load an entire 20GB file into memory at once; I think we need to process part at a time.

Reply:
Changed it to process chunk at a time!